### PR TITLE
task 7 (OOP in js) completed; ESlint was configurated also

### PR DESCRIPTION
**Pull request by mshevko**

### CODE REVIEW

File: feed.js [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+                typeof postsList + ' type provided\n';
+            }
+        }
+        catch(msg) {
+            console.error(msg);
+            return undefined;
+        }
+
+        let failed = postsList.filter(post => !this.add(postsList));
+        return failed;
```
Comment (MaksimViarenich) [inline feed.js:21]: Variable `failed` is not needed.
`return postsList.filter(post => !this.add(postsList));`

File: task7/feed.js [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+        } catch (msg) {
+            console.error(msg);
+            return false;
+        }
+
+        if (feed.validate(post) && (this._postsList.findIndex(elem => elem.id === post.id) === -1)) {
+            this._postsList.push(post);
+            return true;
+        }
+        else {
```
Comment (MaksimViarenich) [inline task7/feed.js:110]: `else` is not needed

File: task7/feed.js [hunk 3]
```diff
@@ -0,0 +1,215 @@
...
+            return false;
+        }
+    }
+
+    edit(id, post) {
+        try {
+            if (typeof id !== 'string') {
+                throw '\nerror [class = feed, func = edit]: string type for \'id\' argument required, ' + typeof id + ' type provided\n';
+            }
```
Comment (MaksimViarenich) [inline task7/feed.js:119]: empty line after `if`

Comment (mshevko) [reply task7/feed.js:119]: i've done some silly thing: i've added these changes to my files while being int the "master" branch. so now i can't checkout "task7" branch from my git bush. can i merge this branch with master ant then commit changes into "task7" directory?

confirm all the changes with the followint picture:
image

and sorry for missing this mistakes due the first time

**Pull request merged**